.github/workflows/validate-page-object-usage.yml

```diff
@@ -2,6 +2,8 @@ name: Validate E2E Page Object usage on modified files
 
 on:
   pull_request:
+    branches:
+      - main
```

Comment:
tweak to avoid running it in the RC branch (or other branches not targetting main)